ssc/vartab.h

```diff
@@ -23,17 +23,18 @@ OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef __lib_vartab_h
 #define __lib_vartab_h
 
-#include "../shared/lib_util.h"
 #include <string>
 #include <vector>
-#include "sscapi.h"
-
 
 #include <unordered_map>
 using std::unordered_map;
 
+#include "../shared/lib_util.h"
+#include "sscapi.h"
```

Comment:
Why so many changes to the core vartab and sscapi of ssc? These will need to be tested in SAM - especially for code generation and parametrics...